Makefile.am

```diff
@@ -21,26 +21,28 @@ libpilgrim_la_SOURCES =
 
 pilgrim_app_generator_SOURCES = 
 pilgrim_app_generator_CFLAGS = $(AM_CFLAGS)
-pilgrim_app_generator_LDFLAGS = -lm -lunwind
+pilgrim_app_generator_LDFLAGS = -lm
 
 pilgrim2text_SOURCES = 
 pilgrim2text_CFLAGS = $(AM_CFLAGS)
-pilgrim2text_LDFLAGS = -lm -lunwind
+pilgrim2text_LDFLAGS = -lm
 
 
 AM_CPPFLAGS = -g -rdynamic
 AM_CFLAGS = -g -rdynamic
 
 
-#ZFP_DIR=/home/wangchen/Sources/zfp-0.5.5/install
-ZFP_DIR=/g/g90/wang116/sources/zfp/install/
-#SZ_DIR=/home/wangchen/Sources/SZ-2.1.12/install
-SZ_DIR=/g/g90/wang116/sources/SZ-2.1.12/install/
+ZFP_DIR=/home/wangchen/Sources/zfp-0.5.5/install
```

Comment:
We should use configure.ac to detect the location of ZFP ro SZ instead of hard code it in Makefile. Can you add `--with-zfp=` and `--with-sz=` as options to configure?

Reply:
Done. I will make all external libraries optional in my next PR.